### DISCUSSION TITLE
First image not required

### DISCUSSION THREAD

**Discussion by SteJW** [spatie/statamic-responsive-images#97] [Ideas]
Is there a reason why the first image is required?

We have created a button group to add an image or a video. Because the first image is required, we're unable to use the button group with choosing a video only. Also for other usages it would help if an image is optional.

In src/Fieldtypes/ResponsiveFields.php line 70 it clearly checks for the first image:
`'required' => $index === 0,`

It would be nice to, if possible, to delete the line.

**Comment by riasvdv** [comment]
This should be fixed in the latest release to only be required when the field is required in the blueprint
